docs/pages/enterprise/sso.mdx

```diff
@@ -4,30 +4,32 @@ description: How to set up single sign-on (SSO) for SSH using Gravitational Tele
 h1: Single Sign-On (SSO) for SSH
 ---
 
-Users of the Enterprise edition of Teleport can login with SSH, Kubernetes, Databases and Web applications
-through a Single Sign-On (SSO) provider used the rest of the organization.
+The Enterprise edition of Teleport, self-hosted or Teleport Cloud, allows for using a organization's
+Single Sign-On (SSO) provider to authenticate and authorize accessing 
+SSH, Kubernetes, Databases, Desktops and Web applications.  Teleport Enterprise supports OIDC and SAML protocols
+for SSO providers in addition to the OpenSource [GitHub SSO](../setup/admin/github-sso.mdx) support.
 
 <TileSet>
   <Tile icon="bolt" title="Azure Active Directory (AD)" href="./sso/azuread.mdx">
-    Configure Azure Active Directory SSO for SSH, Kubernetes, Database and Web apps.
+    Configure Azure Active Directory SSO for SSH, Kubernetes, Database, Desktops and Web apps.
   </Tile>
   <Tile icon="bolt" title="Active Directory (ADFS)" href="./sso/adfs.mdx">
-    Configure Windows Active Directory SSO for SSH, Kubernetes, Database and Web apps.
+    Configure Windows Active Directory SSO for SSH, Kubernetes, Database, Desktops and Web apps.
```

Comment:
I think we should remove all the specific products from these tiles, for two reasons:

1. It goes out of date every time we add a new product.
2. It's technically inaccurate. You don't use SSO for SSH, K8s, Databases, etc. You use SSO for teleport and then you use Teleport certificates for SSH, K8s, Databases, etc